guide/src/class.md

```diff
@@ -17,76 +17,7 @@ struct MyClass {
 The above example generates implementations for [`PyTypeInfo`], [`PyTypeObject`],
 and [`PyClass`] for `MyClass`.
 
-Specifically, the following implementation is generated:
```

Comment:
I moved this code block to the last of the page. 
It's too long for a user who doesn't have a strong interest in the implementation detail.

Reply:
Good idea!